cms/utils/apphook_reload.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals, absolute_import
```

Comment:
We should conform to the rest of the codebase and not use this, ATM (to be clear, I think we should update all the codebase to use this and print_function, but we should be consistent)